### ISSUE TITLE
focal_loss_sigmoid_on_2_classification

### ISSUE THREAD

**Issue by abin24** [hxtkyne/focal-loss-with-tensorflow#1]
103行输入有问题。输入logits 的shape 是4,2 这部分应该没有执行。多分类的56行L没定义
